tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodAttributeExecuteIntegration.cs

```diff
@@ -221,33 +246,33 @@ static void RunRetry(ITestMethod testMethod, TestRunnerState testMethodState, Li
         }
     }
 
-    private static TestStatus HandleTestResult(Test test, ITestMethod testMethod, object? testResultObject, Exception? exception)
+    private static TestStatus HandleTestResult(Test test, ITestMethod testMethod, ITestResult testResult, Exception? exception)
```

Comment:
nit: would it be better to use a `T` here with a generic constraint of `ITestResult` in case we have struct duck types in the future? Maybe not, mostly wondering if we should make that a more general pattern 🤷‍♂️